umbilical/x509.go

```diff
@@ -0,0 +1,85 @@
+// Package umbilical has the temporary logic to back an MTC with an existing
+// X509 certificate chain.
+package umbilical
```

Comment:
Pardon the bikeshed, but what about "bootstrap" as the package name?

Reply:
Bootstrap sounds like something that remains in place 🤔 .

Reply:
Hmm I think of a bootstrap as something that's used initially to get a system going, but then isn't needed anymore. Anyway, plenty of time to argue about the naming later :).